README.md

```diff
@@ -105,7 +105,7 @@ See below the associated versions of Python and RDKit, for which a minor version
 
 | `datamol` | `python`            | `rdkit`                       |
 | --------- | ------------------- | ----------------------------- |
-| `0.10.x`  | `[3.9, 3.10, 3.11]` | `[2022.03, 2022.09]`          |
+| `0.10.x`  | `[3.9, 3.10, 3.11]` | `["2022.09", "2023.03.1"]`    |
```

Comment:
Please leave the `0.10.x` intact and add a new `0.11.x` row?

I usually always bump at `MINOR` whenever a new rdkit version is out.